test/bats/gcp.bats

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bats
+
+load helpers
+
+BATS_TESTS_DIR=test/bats/tests/gcp
+WAIT_TIME=60
+SLEEP_TIME=1
+NAMESPACE=default
+PROVIDER_YAML=https://raw.githubusercontent.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/tam7t/v010/deploy/provider-gcp-plugin.yaml
```

Comment:
Should this be the url be https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/blob/main/deploy/provider-gcp-plugin.yaml?

Reply:
Yeah, this branch has the correct build referenced in the yaml where the `main` branch does not have it yet. I will update this PR once I merge https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/pull/59 and push a new release

Reply:
Updated for the `main` branch.